polling_stations/apps/data_collection/management/commands/import_wokingham.py

```diff
@@ -0,0 +1,36 @@
+"""
+Import Wokingham Polling stations
+"""
+
+from data_collection.management.commands import BaseShpImporter, import_polling_station_shapefiles
+
+class Command(BaseShpImporter):
+    """
+    Imports the Polling Station data from Wokingham Council
+    """
+    council_id     = 'E06000041'
+    districts_name = ''
+    stations_name  = 'POLLING_STATIONS_font_point.shp'
+
+    def import_polling_districts(self):
+        # Because we don't have them :(
+        return 
+
+    def district_record_to_dict(self, record):
+        # return {
+        #     'council': self.council,
+        #     'internal_council_id': record[-1],
+        #     'name': record[0],
+        # }
+        return
+
+    def station_record_to_dict(self, record):
+        return {
+            'council'            : self.council,
+            'internal_council_id': " ".join(record),
+            'postcode'           : '(No postcode supplied)',
```

Comment:
Worth geocoding at point of import?  Google's reverse geocode is good enough most of the time.